java/ql/src/Security/CWE/CWE-200/SpringBootActuators.qhelp

```diff
@@ -0,0 +1,36 @@
+<!DOCTYPE qhelp PUBLIC
+  "-//Semmle//qhelp//EN"
+  "qhelp.dtd">
+<qhelp>
+<overview>
+<p>Spring Boot includes features called actuators that let you monitor and interact with your
+web application. Exposing unprotected actuator endpoints can lead to information disclosure or
+even to remote code execution.</p>
+</overview>
+
+<recommendation>
+<p>Since actuator endpoints may contain sensitive information, carefully consider when to expose them,
+and secure them as you would any sensitive URL. Actuators are secured by default when using Spring
+Security without a custom configuration. If you wish to define a custom security configuration,
+consider only allowing users with certain roles access to the endpoints.
```

Comment:
I understood what you meant but perhaps we should slightly tweak this to improve the parsing of the last part of the sentence.
```suggestion
consider only allowing users with certain roles to access these endpoints.
```